src/Tizen.NUI/src/internal/WebView/WebHttpRequestInterceptor.cs

```diff
@@ -82,20 +113,64 @@ public bool SetResponseStatus(int statusCode, string customStatusText)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public bool AddResponseHeader(string fieldName, string fieldValue)
         {
-            bool result = Interop.WebHttpRequestInterceptor.AddResponseHeader(SwigCPtr, fieldName, fieldValue);
+            bool result = Interop.WebHttpRequestInterceptor.AddResponseHeader(interceptorHandle, fieldName, fieldValue);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return result;
+        }
+
+        /// <summary>
+        /// Adds HTTP headers to response for intercepted request.
+        /// <param name="headers">Map of response headers</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool AddResponseHeaders(PropertyMap headers)
+        {
+            bool result = Interop.WebHttpRequestInterceptor.AddResponseHeaders(interceptorHandle, PropertyMap.getCPtr(headers));
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             return result;
         }
 
         /// <summary>
         /// Writes whole response body at once.
         /// <param name="body">Contents of response</param>
-        /// <param name="length">Length of Contents of response</param>
+        /// <param name="length">Length of contents of response</param>
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public bool AddResponseBody(string body, uint length)
         {
-            bool result = Interop.WebHttpRequestInterceptor.AddResponseBody(SwigCPtr, body, length);
+            bool result = Interop.WebHttpRequestInterceptor.AddResponseBody(interceptorHandle, body, length);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return result;
+        }
+
+        /// <summary>
+        /// Writes whole response body at once.
+        /// <param name="headers">Headers of response</param>
+        /// <param name="body">Contents of response</param>
+        /// <param name="length">Length of contents of response</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool AddResponse(string headers, string body, uint length)
```

Comment:
Ditto. I think `Length` is not required.

Reply:
As I commented above, the API `ewk_intercept_request_response_set` and `ewk_intercept_request_response_body_set` are the finisher.
So I think using original word `Set` is more suitable than `Add` here.

* AddResponseBody -> SetResponseBody
* AddResponse -> SetResponse